app/models/user.rb

```diff
@@ -6,16 +6,16 @@ class User < ApplicationRecord
   validates :last_name, presence: true
   validates :stem_achiever_contact_no, presence: true
   validates :stem_credentials_access_token, presence: true
-  validates :stem_credentials_refresh_token, presence: true
+  # validates :stem_credentials_refresh_token, presence: true
```

Comment:
Should we still be validating the refresh token here?

Reply:
We no longer get a refresh token. It used to come with the login, we stored it, and never used it.

I had commented it out during testing, will just delete